products/paas/build-deploy.md

```diff
@@ -45,3 +45,15 @@ The installer will create an administrator account with the default credentials.
 | `admin` | `shopware` |
 
 Make sure to change this password immediately in your Administration account settings. Not doing so is a security risk.
+
+## Composer authentication
+
+To install Extensions from the Shopware Store via composer, you will have to authenticate yourself. In your local development environment this is possible by creating an `auth.json` file which contains your auth token. However, this file should not be commited into the repository. 
+
+The following command adds your authentication token to the secure environment variable storage of Shopware Paas. The variable, which contains the content which would otherwise be in `auth.json`, will be available during the build step and be automatically picked up by Composer.
```

Comment:
```suggestion
The following command adds your authentication token to the secure environment variable storage of Shopware Paas. This variable (contains the content which would otherwise be in `auth.json`) will be available during the build step and be automatically picked up by the composer.
```